solutions/core-landing-zone/lz-folder/audits/logging-project/cloud-storage-buckets.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Cloud Storage bucket to store logs related to security incidents
+# https://cloud.google.com/logging/docs/routing/copy-logs
+# AU-9, AU-11 - Storage bucket created to hold logs related to security incidents (AU-11). Log is protected from modification and deletion (AU-9)
+apiVersion: storage.cnrm.cloud.google.com/v1beta1
+kind: StorageBucket
+metadata:
+  name: security-incident-log-bucket # kpt-set: ${security-incident-log-bucket}
```

Comment:
I think the `security-incident-log-bucket` setter is missing?  (it'll be worth noting that this name must be unique across all GCP...so maybe we use the logging-project-id somewhere in the name since that's unique as well?)

Reply:
Yes, it is. I will add it. Thanks!

Reply:
Addressed, please review and resolve if satisfied.